### PR TITLE
fix: [CDS-86821]: document agent restart requirement for byo argo upg…

### PR DESCRIPTION
**Pull request by ashinsabu3**
…rade

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md [hunk 1]
```diff
@@ -213,4 +213,4 @@ When you have completed [setting up the cluster](/docs/continuous-delivery/gitop
 
 * Harness honors Argo CD project permissions. If the project selected for the Harness application does not have permissions for the repository or cluster, then Harness returns a permissions-related error. You must go to Argo CD and adjust the project's scoped repositories and destinations.
 * A non-BYOA setup does not support mapping multiple Argo CD projects to a single Harness project.
-
+* **\[IMPORTANT\]Upgrading your Argo CD Version**: If you upgrade your Argo CD version to any version that is >= **v2.8.0** with a BYO Argo Gitops agent installed, you will have to restart the agent pods(by restarting the deployment/statefulset) in order for the agent to apply required configurations.
```
Comment (mankrit-singh) [inline docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md:216]: let's exclude this - [IMPORTANT\] 

also just restarting the GitOps agent pod works right?

Comment (ashinsabu3) [reply docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md:216]: Sure.

What if it's an HA agent?

**Review by mteodor — DISMISSED** [review]

**Review by mankrit-singh — DISMISSED** [review]

File: docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md [hunk 2]
```diff
@@ -213,4 +213,4 @@ When you have completed [setting up the cluster](/docs/continuous-delivery/gitop
 
 * Harness honors Argo CD project permissions. If the project selected for the Harness application does not have permissions for the repository or cluster, then Harness returns a permissions-related error. You must go to Argo CD and adjust the project's scoped repositories and destinations.
 * A non-BYOA setup does not support mapping multiple Argo CD projects to a single Harness project.
-
+* **Upgrading your Argo CD Version**: If you upgrade your Argo CD version to any version that is >= **v2.8.0** with a BYO-Argo Gitops agent installed, you will have to restart the agent pods(by restarting the deployment or the pods individually) in order for agent to pickup required configuration change.
```
Comment (michaelcretzman) [inline docs/continuous-delivery/gitops/connect-and-manage/multiple-argo-to-single-harness.md:216]:
```suggestion
* **Upgrading your Argo CD Version**: If you upgrade your Argo CD version to any version that is >= v2.8.0 with a BYO Argo GitOps agent installed, you will have to restart the agent pods in order for the agent to pickup the required configuration change. You canrestart the deployment or the pods individually.
```

**Review by michaelcretzman — APPROVED** [review]
Approved with edits

**Pull request merged**